src/lib/azure-importer/index.ts

```diff
@@ -371,6 +371,11 @@ export class AzureImporterModel implements IOutputModelInterface {
     };
   }
 
+  /**
+   * Calculates number of seconds covered by each individual input using azure-time-window value
+   * @param params
+   * @returns number
```

Comment:
these params, return things are actual for plain js, but with typescript we can skip them

Reply:
fixed